### DISCUSSION TITLE
Fronting domain خطایه

### DISCUSSION THREAD

**Discussion by amazon-mix** [MortezaBashsiz/CFScanner#312] [General]
سلام وقتی توی کانفینگ json ورکر ایدی وارد میکنم میرم توی برنامه با خطایی Fronting domain is not accessible! you might need to get new fronting url from our github or check your Internet connection. رو به رو میشم ولی اگه با فیلتر شکن برم درست میشه ولی ایپی رنج نشون نمیده چیکار کنم ؟

**Comment by AliKh-A** [comment]
سلام منم دقیقا به این ارور برخوردم جدیدا

**Comment by f4r3ingh75** [comment]
سلام منم این مشکل رو دارم لطفا راهنمایی کنید.
